core/node/node.go

```diff
@@ -66,7 +66,7 @@ func (node *Node) Start() error {
 		}
 	}()
 
-	originalLocation, err := node.locationCache.DetectLocation()
+	originalLocation, err := node.locationCache.GetOrigin()
```

Comment:
What was wrong with this interface which clearly specified that original country is being set here:
```
// Cache allows caching location
type Cache interface {
	Get() Location
	RefreshAndGet() (Location, error)
}
```

Reply:
Cache.Get is not discernible